tck-dist/src/main/starter/ee-pom.xml

```diff
@@ -33,9 +33,9 @@
   <maven.compiler.testRelease>17</maven.compiler.testRelease>
 
   <!-- Dependency and Plugin Versions -->
-  
-  <!-- Default - use local snapshot - can be overwritten by runner -->
-  <jakarta.data.version>1.0.0-SNAPSHOT</jakarta.data.version>
+  <jakarta.data.version>1.0.0</jakarta.data.version>
```

Comment:
note that the data-api version and data-tck version are two different things having different lifecycles. Accepting a TCK challenge will require new TCK version while keeping the API version untouched